AppService/python-webapp-on-azure.yml

```diff
@@ -34,27 +33,15 @@ jobs:
   - name: python install
     working-directory: ${{ env.WORKING_DIRECTORY }}
     run: |
-     sudo apt install python${{ env.PYTHON_VERSION }}-venv
      python -m venv --copies antenv
      source antenv/bin/activate
      pip install setuptools
      pip install -r requirements.txt
-  # Azure login
-  - uses: azure/login@v1
```

Comment:
any specific reason we are removing login/logout actions from the sample?

Reply:
Wanted to keep it consistent with that of nodejs sample and keep it simple. When we use publish profile, login and logout are not needed.